src/app/core/services/http/http.service.ts

```diff
@@ -34,7 +33,7 @@ export class HttpService {
     private translate: TranslateService,
     private alert: AlertController,
   ) {
-    this.baseUrl = environment.baseUrl;
+    this.baseUrl = window['env']['baseUrl'];
```

Comment:
Why are we making this change in our repo?
For us it will be environment.baseUrl right?